web/themes/custom/pineapple/templates/views/views-view--author-content.html.twig

```diff
@@ -0,0 +1,95 @@
+{#
```

Comment:
General comment about these view templates, you now have 4 different view templates which are essentially similar because you're only adding a few classes on a couple of divs. After this task we should consider creating a base view file which will act as a component that has attributes for the divs you're adding classes to and then you can include the file as you normally do for your components and add the classes in that manner, this will drastically reduce markup and make it far clearer what changes are actually being made in each view file because as it is, if you're ensure of what you're looking for it won't be clear what's changed from the base view file. So this one to add to the todo list.

Reply:
alright, thank you